packages/amplication-server/src/core/block/block.service.ts

```diff
@@ -942,4 +942,35 @@ export class BlockService {
       },
     });
   }
+
+  async getLatestVersions(args: {
+    where: Prisma.BlockWhereInput;
+  }): Promise<Block[]> {
+    const blocks = await this.prisma.block.findMany({
+      where: {
+        ...args.where,
+        resourceId: args.where.resourceId,
+        deletedAt: null,
+      },
+      include: {
+        versions: {
+          where: {
+            versionNumber: {
+              not: CURRENT_VERSION_NUMBER,
```

Comment:
```suggestion
              gt: CURRENT_VERSION_NUMBER,
```

Reply:
CURRENT_VERSION_NUMBER is 0 - so both will give the same results.
we are using not in entities so i prefer to keep the same code
the latest version will always be larger then zero as long as there is a version - so the filter is actually "give me top 1 version from all published versions"